config/params/mainnet_config.go

```diff
@@ -23,7 +23,10 @@ const (
 	mainnetAltairForkEpoch = 74240 // Oct 27, 2021, 10:56:23am UTC
 	// Bellatrix Fork Epoch for mainnet config.
 	mainnetBellatrixForkEpoch = 144896 // Sept 6, 2022, 11:34:47am UTC
-	mainnetDenebForkEpoch     = math.MaxUint64
+	// Capella Fork Epoch for mainnet config.
+	mainnetCapellaForkEpoch = 194048 // April 12, 2023, 22:27:35 UTC
```

Comment:
That being said, for consistency or at least appearing the same as the above comments, it should be: 
```suggestion
	mainnetCapellaForkEpoch = 194048 // April 12, 2023, 22:27:35pm UTC
```